rust/crd/src/lib.rs

```diff
@@ -367,15 +367,15 @@ pub struct NifiStorageConfig {
 
 impl NifiCluster {
     /// The name of the role-level load-balanced Kubernetes `Service`
-    pub fn node_role_service_name(&self) -> Option<String> {
-        self.metadata.name.clone()
+    pub fn node_role_service_name(&self) -> String {
+        self.name_unchecked()
```

Comment:
I prefer the name_any to not panic the operator.
```suggestion
        self.name_any()
```